ruma-events/src/room/message.rs

```diff
@@ -157,8 +152,34 @@ impl MessageEventContent {
     }
 }
 
+impl<'de> de::Deserialize<'de> for MessageEventContent {
```

Comment:
Any reason you didn't go with my suggestion from #406 to put this into a sub-module?

Reply:
No reason really, I intended to factor it out into a new module once I got everything else done, in any case, I made a new module with the latest commit.